systemSmartContractsConfig.toml

```diff
@@ -1,22 +1,23 @@
 [StakingSystemSCConfig]
     GenesisNodePrice = "2500000000000000000000" #2.5K eGLD
     MinStakeValue = "100000000000000000000" #100 eGLD
+    MinUnstakeTokensValue = "10000000000000000000" #10eGLD
     UnBondPeriod = 2500
     MinStepValue = "100000000000000000000"
-    AuctionEnableEpoch = 1000000
     StakeEnableEpoch = 0
+    StakingV2Epoch = 14
+    DoubleKeyProtectionEnableEpoch = 14
     NumRoundsWithoutBleed = 100
     MaximumPercentageToBleed = 0.5
     BleedPercentagePerRound = 0.00001
     MaxNumberOfNodesForStake = 2169
```

Comment:
3200 ?

Reply:
this should be kept at 2169. It is only used in genesis. After that, it will be used from config.toml.

Reply:
Ok